### DISCUSSION TITLE
感觉文档对新手不友好

### DISCUSSION THREAD

**Discussion by gaoyusongcn** [Zhengqbbb/cz-git#63] [Q&A]
对于 [@快速开始](https://cz-git.qbb.sh/zh/guide/#%E9%A1%B9%E7%9B%AE%E4%B8%AD%E4%BD%BF%E7%94%A8) 这里掺杂了 [@commitlint/config-conventional](https://github.com/conventional-changelog/commitlint/blob/master/@commitlint/config-conventional/index.js) 和 [@cz-git 的 Javascript 模板](https://cz-git.qbb.sh/zh/config/#javascript-%E6%A8%A1%E6%9D%BF)，两个配置我搞混了。

不想全局 `npm install -g commitizen` 想加在 `package.json` 里面。

**Comment by Zhengqbbb** [comment]
> 对于 [@快速开始](https://cz-git.qbb.sh/zh/guide/#%E9%A1%B9%E7%9B%AE%E4%B8%AD%E4%BD%BF%E7%94%A8) 这里掺杂了 [@commitlint/config-conventional](https://github.com/conventional-changelog/commitlint/blob/master/@commitlint/config-conventional/index.js) 和 [@cz-git 的 Javascript 模板](https://cz-git.qbb.sh/zh/config/#javascript-%E6%A8%A1%E6%9D%BF)，两个配置我搞混了。

可恶 有没有推荐一点的建议呢 🤔 

> 不想全局 npm install -g commitizen 想加在 package.json 里面。

如果是这样 建议直接使用 `czg` [czg](https://cz-git.qbb.sh/zh/cli/install.html#%E9%A1%B9%E7%9B%AE%E4%B8%AD%E4%BD%BF%E7%94%A8)
1. 一步到位，无需其他操作(就是因为 commitizen 的设计安装太繁杂)
2. 最关键是 `commitizen` 依赖太多了，整体下载到node_modules太大，下载依赖慢，而且因为设计导致的启动速度太慢了
```
$ npm install -D commitizen
added 148 packages, and audited 149 packages in 2 m
$ du -sh ./node_modules
102M ./node_modules

$ npm install -D czg
added 1 package, and audited 2 packages in 408 ms
$ du -sh ./node_modules/*
1.3M ./node_modules/czg
```
